apps/api/src/app/message-queue/message-queue.service.ts

```diff
@@ -137,6 +142,51 @@ export class MessageQueueService implements OnModuleInit, OnModuleDestroy {
 
       await sendMail(from, to, subject, mailTemplate);
     }
+    else if (json_message.run_status === 'partial_success') {
```

Comment:
Can you create an enum class for the run_status?

Reply:
yes. will do.